test/owners.js

```diff
@@ -1,18 +1,33 @@
-const chai = require('chai')
-const expect = chai.expect
+const { expect } = require('chai')
 const nockHelper = require('./helpers/nock_helper')
-
 const Hubspot = require('..')
-const hubspot = new Hubspot({ apiKey: 'demo' })
 
 describe('Owners', function() {
-  before(nockHelper.mockEndpoint('/owners/v2/owners', []))
-  after(nockHelper.resetNock)
   describe('get', function() {
+    before(nockHelper.mockEndpoint('/owners/v2/owners', []))
+    after(nockHelper.resetNock)
+
     it('Should return all owners', function() {
+      const hubspot = new Hubspot({ apiKey: 'demo' })
```

Comment:
In the event `hubspot.owners.get()` hits an endpoint other than the mocked endpoint, what happens? Is there value in asserting `hubspot.owners.get()` issued a request to the expected endpoint with the expected params (if any)?

Reply:
The way I test this is by running the test suite with this mocking turned off. We can run tests without nock by running `NOCK_OFF=true npm run mocha`.

I'd like to add a section for contributing suggesting that you initially write tests without nock. Once everything is passing it's then worth adding the `before` and `after` hooks to mock out the actual API request.